A Task on Python - Exceptions
